components/kflogin/README.md

```diff
@@ -0,0 +1,32 @@
+## Basic Auth Login Page
+
+UI to login kubeflow cluster.
+
+#### This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `npm start`
+
+Runs the app in the development mode.<br>
+Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+
+The page will reload if you make edits.<br>
+You will also see any lint errors in the console.
+
+### `npm test`
+
+Launches the test runner in the interactive watch mode.<br>
+See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+
+### `npm run build`
```

Comment:
Should we cleanup this readme? If there are no actual tests (or we're not planning to run them), let's remove the tests section, and we can change the build section below to mention using the makefile instead?